tests/conftest.py

```diff
@@ -111,3 +111,24 @@ def models_s3_bucket_endpoint(pytestconfig: pytest.Config) -> str:
             "Either pass with `--models-s3-bucket-endpoint` or set `MODELS_S3_BUCKET_ENDPOINT` environment variable"
         )
     return models_bucket_endpoint
+
+
+@pytest.fixture(scope="session")
+def supported_accelerator_type(pytestconfig: pytest.Config) -> str:
+    accelerator_type = pytestconfig.option.supported_accelerator_type
+    if not accelerator_type:
+        return None
+    if accelerator_type.lower() not in ["nvidia", "amd", "gaudi"]:
```

Comment:
Maybe create a class `AcceleratorType` in constants.py, and reuse across the PR e.g. `AcceleratorType.NVIDIA`

Reply:
This doesn't seem ideal, as the accelerator type is a variable that  stores the output based on the CLI argument or environment variable. However, I have modified the code to check a constant variable stored in constant for better clarity and understanding.